tutor/src/models/reference-book/page.js

```diff
@@ -8,15 +8,17 @@ import ChapterSection from '../chapter-section';
 import { MediaActions } from '../../flux/media';
 
 const NON_ASSIGNABLE_TITLES = [
-  'Visual Connection Questions',
-  'Review Questions',
-  'Critical Thinking Questions',
-  'Conceptual Questions',
-  'Problems & Exercises',
+  'Glossary',
   'Key Terms',
   'Chapter Summary',
-  'Glossary',
   'Section Summary',
+  'Review Questions',
+  'Conceptual Questions',
+  'Problems & Exercises',
+  'Test Prep for AP® Courses',
+  'Test Prep for AP® Courses',
```

Comment:
Dupe. From your description I think one of them is supposed to be `"Science Practice Challenge Questions"`